docs/main_text.md

```diff
@@ -534,15 +534,24 @@ Below is the detailed folder structure:
+Example analyses are designed to guide you through obtaining data from the refine.bio web interface and provide instruction for adapting the analysis for a dataset of your choice.
+View our <a href = "https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html" target = "blank">Getting Started page</a>, <a href = "https://alexslemonade.github.io/refinebio-examples/02-microarray/00-intro-to-microarray.html" target = "blank">Introduction to Microarray</a>, or <a href = "https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html" target = "blank">Introduction to RNA-seq</a> to start using refine.bio examples.
 
-* Differential expression analysis [<a href = "https://github.com/AlexsLemonade/refinebio-examples/tree/v1.0/differential-expression" target = "blank">README</a>, <a href = "https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/v1.0/differential-expression/microarray_DGE.nb.html" target = "blank">microarray notebook</a>, <a href = "https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/v1.0/differential-expression/rnaseq_DGE.nb.html" target = "blank">RNA-seq notebook</a>]
-* Converting between different gene identifiers [<a href = "https://github.com/AlexsLemonade/refinebio-examples/tree/v1.0/ensembl-id-convert" target = "blank">README</a>, <a href = "https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/v1.0/ensembl-id-convert/ensembl_id_convert.nb.html" target = "blank">notebook</a>]
-* Ortholog mapping [<a href = "https://github.com/AlexsLemonade/refinebio-examples/tree/v1.0/ortholog-mapping" target = "blank">README</a>, <a href = "https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/v1.0/ortholog-mapping/ortholog_mapping_example.nb.html" target = "blank">notebook</a>]
-* Clustering/heatmap generation [<a href = "https://github.com/AlexsLemonade/refinebio-examples/tree/v1.0/clustering" target = "blank">README</a>, <a href = "https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/v1.0/clustering/clustering_example.nb.html" target = "blank">notebook</a>]
+Here's a sneak peek of examples that are available:
+
+* <a href = "https://alexslemonade.github.io/refinebio-examples/02-microarray/differential-expression_microarray_02_several-groups.html" target = "blank">Differential gene expression analysis for several groups (microarray)</a>
```

Comment:
```suggestion
* <a href = "https://alexslemonade.github.io/refinebio-examples/02-microarray/differential-expression_microarray_02_several-groups.html" target = "blank">Differential gene expression analysis for several groups (Microarray)</a>
```

Reply:
I'm slightly concerned about linking the more complicated "several groups" instead of the "2 groups" one. But not so much that I think we need to for sure change it.

Reply:
What if I added the 2 groups for RNA-seq, too?

Reply:
Sounds good to me!